Task3/src/main/java/controller/ConsoleController.java

```diff
@@ -0,0 +1,88 @@
+package controller;
+
+import model.House;
+import model.HouseManager;
+import view.ConsoleView;
+import view.View;
+
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.Scanner;
+
+public class ConsoleController implements Controller {
+    String command;
+    Scanner scanner = new Scanner(System.in);
+    View view = new ConsoleView();
+    HouseManager houseManager = new HouseManager();
+
+    public void execute() {
+        do {
+            view.printMenu();
+            ((ConsoleView) view).printQuestion();
```

Comment:
If you add printQuestion method to View interface you would able to get rid of downcast View to ConsoleView